.github/ISSUE_TEMPLATE/3-help.md

```diff
@@ -0,0 +1,14 @@
+---
+name: "⁉️ Need help with Apollo Voyager Server?"
+about: Please file an issue in our repo.
+
+---
+
+
+If you have a question about Apollo Voyager Server that is not a bug report or feature
+request, feel free to post it here.
+
+You can also reach the aerogear team at [#aerogear](ircs://chat.freenode.net:6697/aerogear) on [Freenode IRC](https://freenode.net/) or the 
+[aerogear google group](https://groups.google.com/forum/#!forum/aerogear).
+
```

Comment:
IMHO it is a bug and/or RFE and/or a question should be defined by a label/tag. Users/contributors/QA are not usually able to the category this info. It is common they believe that an RFE is an issue for example.